src/components/LayerControl/LayerControl.module.css

```diff
@@ -33,6 +33,7 @@
   justify-content: center;
   gap: var(--unit);
   user-select: none;
+  margin-right: var(--half-unit);
```

Comment:
There's no need to add padding to these controls right now, we're only interested in MCDA layer editing in this ticket

Reply:
..on the other hand, it fixes alignment with Deselect button texts. Let it be :)